src/backend/src/services/design-review.services.ts

```diff
@@ -45,6 +52,135 @@ export default class DesignReviewService {
     return designReviewTransformer(deletedDesignReview);
   }
 
+  /**
+   * Creates a design review
+   * @param submitter user who submitted the design review
+   * @param dateScheduled when the design review is scheduled for
+   * @param teamTypeId team type id of the design review
+   * @param requiredMemberIds ids of the required members to attend the design review
+   * @param optionalMemberIds ids of the optional members to attend the design reivew
+   * @param isOnline if design review is online
+   * @param isInPerson if design review is in person
+   * @param docTemplateLink link to the doc template
+   * @param wbsNum wbs number for the design review
+   * @param meetingTimes the meeting times for the design review
+   * @param zoomLink link for the zoom if design review is online
+   * @param location location of the design review if in person
+   * @returns a design review
+   */
+  static async createDesignReview(
+    submitter: User,
+    dateScheduled: Date,
+    teamTypeId: string,
+    requiredMemberIds: number[],
+    optionalMemberIds: number[],
+    isOnline: boolean,
+    isInPerson: boolean,
+    docTemplateLink: string,
+    wbsNum: WbsNumber,
+    meetingTimes: number[],
+    zoomLink?: string,
+    location?: string
+  ): Promise<DesignReview> {
+    if (!isLeadership(submitter.role)) throw new AccessDeniedException('create design review');
+
+    const teamType = await prisma.teamType.findFirst({
+      where: { teamTypeId }
+    });
+
+    if (!teamType) {
+      throw new NotFoundException('Team Type', teamTypeId);
+    }
+
+    const wbsElement = await prisma.wBS_Element.findUnique({
+      where: {
+        wbsNumber: {
+          carNumber: wbsNum.carNumber,
+          projectNumber: wbsNum.projectNumber,
+          workPackageNumber: wbsNum.workPackageNumber
+        }
+      }
+    });
+
+    if (!wbsElement) {
+      throw new NotFoundException('WBS Element', wbsNum.carNumber);
+    }
+
+    // checks if the meeting times are valid times and are all continous (ie. [1, 2, 3, 4])
+    for (let i = 0; i < meetingTimes.length - 1; i++) {
```

Comment:
if there's only 1 element, the for loop doesn't check correctly. I'd go from 0 to i < meetingTimes.length, then add a check that only checks the meeting range (0-83) and not the continuousness if you're on the last element